addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -83,13 +83,13 @@ def _check_before_xml_exporting(self):
             raise UserError(_("%s must have a country") % (seller.display_name))
 
         # <1.1.1.2>
-        if not seller.vat:
+        if not seller.vat and seller.country_id.code == 'IT':
```

Comment:
It is only Italy that is exporting though.  It would be weird not to be the case.